test/test-integration/src/test/java/com/alipay/sofa/rpc/test/warmup/WarmUpTest.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.rpc.test.warmup;
+
+import com.alipay.sofa.rpc.client.ProviderInfoAttrs;
+import com.alipay.sofa.rpc.common.RpcConstants;
+import com.alipay.sofa.rpc.config.ConsumerConfig;
+import com.alipay.sofa.rpc.config.ProviderConfig;
+import com.alipay.sofa.rpc.config.RegistryConfig;
+import com.alipay.sofa.rpc.config.ServerConfig;
+import com.alipay.sofa.rpc.context.RpcInternalContext;
+import com.alipay.sofa.rpc.context.RpcInvokeContext;
+import com.alipay.sofa.rpc.context.RpcRuntimeContext;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+
+/**
+ *
+ * @author <a href="mailto:lw111072@antfin.com">LiWei.Liangen</a>
+ * @version $Id: warmUpTest.java, v 0.1 2018年04月23日 上午10:20 LiWei.Liangen Exp $
+ */
+public class WarmUpTest {
+
+    @After
+    public void after() {
+        RpcRuntimeContext.destroy();
+        RpcInternalContext.removeContext();
+        RpcInvokeContext.removeContext();
+    }
+
+    @Test
+    public void testWarmUp() throws InterruptedException, NoSuchFieldException, IllegalAccessException {
+
+        RegistryConfig registryConfig = new RegistryConfig()
+            .setProtocol("zookeeper")
```

Comment:
If you want to run integration test with zookeeper, move test case to module `test-integration-3rd`.